invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -28,22 +28,20 @@ const RecordVersionItem = ({ item, activeVersion }) => {
     <>
       <Grid.Row
         key={item.id}
-        columns={2}
+        columns={1}
         {...(activeVersion && { className: "version-active" })}
       >
-        <Grid.Column width={10}>
+        <Grid.Column>
+        <small className="text-muted" style={{ float: "right" }}>
+             {item.publication_date}
+           </small>
           <a href={`/records/${item.id}`}>Version {item.version}</a>
-          {doi && (
-            <small className="text-muted" style={{ float: "left" }}>
-              {doi}
+          {<br/>}{doi && (
+            <small  className="text-muted">
+              {doi} 
```

Comment:
extra space 😅 do you use vs code? you can configure it to remove them automatically on-save 👼

Reply:
ah yes its because of the new laptop, dont have the extensions installed yet haha. thank you Pablo!